src/e2e/e2e_protection.cpp

```
@@ -79,7 +78,7 @@ Result E2EProtection::validate(const Message& message, const E2EConfig& config)
     // Call profile's validate method
     // Note: validate is const, but we need non-const message for some operations
```

Cppcheck (2.20.0)

[style] The function 'set_request_id' is never used.

(unusedFunction)